app/javascript/packs/clipboardCopy.js

```diff
@@ -1,5 +1,24 @@
-document.addEventListener('DOMContentLoaded', () => {
-  window.WebComponents.waitFor(() => {
-    import('@github/clipboard-copy-element');
-  });
+HTMLDocument.prototype.ready = new Promise(resolve => {
```

Comment:
This is a side note for another PR, but we have this same code in several files. We should only load it once as it will apply to all JS on the page.